x-pack/plugins/stack_connectors/public/connector_types/cases_webhook/steps/update.tsx

```diff
@@ -23,185 +23,203 @@ interface Props {
+                    validator: emptyField(i18n.UPDATE_METHOD_REQUIRED),
+                  },
+                ],
+              }}
+              css={styles.method}
+              componentProps={{
+                euiFieldProps: {
+                  'data-test-subj': 'webhookUpdateMethodSelect',
+                  options: HTTP_VERBS.map((verb) => ({ text: verb.toUpperCase(), value: verb })),
+                  readOnly,
+                },
+              }}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <UseField
+              path="config.updateIncidentUrl"
+              config={{
+                label: i18n.UPDATE_INCIDENT_URL,
+                validations: [
+                  {
+                    validator: urlField(i18n.UPDATE_URL_REQUIRED),
+                  },
+                ],
+                helpText: i18n.UPDATE_INCIDENT_URL_HELP,
+              }}
+              component={MustacheTextFieldWrapper}
+              componentProps={{
+                euiFieldProps: {
+                  readOnly,
+                  'data-test-subj': 'webhookUpdateUrlText',
+                  messageVariables: urlVars,
+                  paramsProperty: 'updateIncidentUrl',
+                  buttonTitle: i18n.ADD_CASES_VARIABLE,
+                  showButtonTitle: true,
+                },
+              }}
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <UseField
+              path="config.updateIncidentJson"
+              config={{
+                helpText: i18n.UPDATE_INCIDENT_JSON_HELP,
+                label: i18n.UPDATE_INCIDENT_JSON,
+                validations: [
+                  {
+                    validator: emptyField(i18n.UPDATE_INCIDENT_REQUIRED),
+                  },
+                  {
+                    validator: containsTitleAndDesc(),
+                  },
+                ],
+              }}
+              component={JsonFieldWrapper}
+              componentProps={{
+                euiCodeEditorProps: {
+                  height: '200px',
+                  isReadOnly: readOnly,
+                  ['aria-label']: i18n.CODE_EDITOR,
+                },
+                dataTestSubj: 'webhookUpdateIncidentJson',
+                messageVariables: [...casesVars, ...urlVars],
+                paramsProperty: 'updateIncidentJson',
+                buttonTitle: i18n.ADD_CASES_VARIABLE,
+                showButtonTitle: true,
+              }}
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+        <EuiSpacer size="m" />
+        <EuiSwitch
+          label={i18n.STEP_4B}
+          showLabel={true}
+          onChange={onAddCommentToggle}
+          checked={isAddCommentToggled}
+          data-test-subj="webhookAddCommentToggle"
         />
-      </EuiFlexItem>
-    </EuiFlexGroup>
-  </span>
-);
+        {isAddCommentToggled && (
+          <>
+            <EuiSpacer size="s" />
```

Comment:
This spacer felt too small imo.

```suggestion
            <EuiSpacer size="m" />
```

But let's see how things look like when you add the missing text I mentioned in the other comment